src/appmixer/google/gmail/ReplyToEmail/ReplyToEmail.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+const GoogleApi = require('googleapis');
+const commons = require('../../google-commons');
+const mailcomposer = require('mailcomposer');
+const Promise = require('bluebird');
+
+// GoogleApi initialization & promisify of some API functions for convenience
+const gmail = GoogleApi.gmail('v1');
+const send = Promise.promisify(gmail.users.messages.send, { context: gmail.users.messages });
+const modify = Promise.promisify(gmail.users.messages.modify, { context: gmail.users.messages });
+
+/**
+ * GMail send email component.
+ * @extends {Component}
+ */
+module.exports = {
+
+    async receive(context) {
+
+        /**
+         * @type MailComposer
+         */
+        let mail = Object.assign({}, context.messages.in.content);
+        if (mail.from && mail.sender) {
+            mail.from = `${mail.sender} <${mail.from}>`;
+        } else if (mail.sender) {
+            mail.from = `${mail.sender} <${context.profileInfo.email}>`;
+        }
+
+        if (context.messages.in.content.cc) {
+            mail.cc = context.messages.in.content.cc;
+        }
+
+        if (context.messages.in.content.bcc) {
+            mail.bcc = context.messages.in.content.bcc;
+        }
+
+        const { attachments = {} } = context.messages.in.content;
+        const fileIds = (attachments.ADD || [])
+            .map(attachment => (attachment.fileId || null))
+            .filter(fileId => fileId !== null);
+
+        mail.attachments = await Promise.map(fileIds, async (fileId) => {
+            const fileInfo = await context.getFileInfo(fileId);
+            const fileStream = await context.getFileReadStream(fileId);
+            return { filename: fileInfo.filename, content: fileStream };
+        });
+
+        // Add signature to the email content if provided
+        if (context.messages.in.content.signature) {
+            if (mail.html) {
+                mail.html += `<br><br>${context.messages.in.content.signature}`;
+            } else if (mail.text) {
+                mail.html = `${mail.text.replace(/\n/g, '<br>')}<br><br>${context.messages.in.content.signature}`;
+                delete mail.text;
+            } else {
+                mail.html = context.messages.in.content.signature;
+            }
+        }
+
+        if (context.messages.in.content.thread) {
+            mail.threadId = context.messages.in.content.thread;
+        }
+
+        return new Promise((resolve, reject) => {
```

Comment:
it seems the code is the same as in the CreateDraft, could it be moved to the gmail-commons?